test/integration/idle_timeout_integration_test.cc

```diff
@@ -178,8 +193,94 @@ TEST_P(IdleTimeoutIntegrationTest, PerStreamIdleTimeoutAfterBidiData) {
 
 // Successful request/response when per-stream idle timeout is configured.
 TEST_P(IdleTimeoutIntegrationTest, PerStreamIdleTimeoutRequestAndResponse) {
+  enable_per_stream_idle_timeout_ = true;
+  testRouterRequestAndResponseWithBody(1024, 1024, false, nullptr);
+}
+
+TEST_P(IdleTimeoutIntegrationTest, RequestTimeoutConfiguredRequestResponse) {
+  enable_request_timeout_ = true;
+  testRouterHeaderOnlyRequestAndResponse(true, nullptr);
```

Comment:
Why testRouterHeaderOnly here and including body above?

Reply:
Test with body is below, the diff doesn't highlight it bc git.